### ISSUE TITLE
add labels to docker image

### ISSUE THREAD

**Issue by itaysk** [aquasecurity/tracee#391]
goreleaser used to add labels in the built container image. now that we don't use it anymore we should research which labels are customary and add to the dockerfile. for example: `LABEL org.label-schema.name=tracee`

**Comment by krol3** [comment]
We have this default labels using GitHub action 
```
"Labels": {
                "org.opencontainers.image.created": "2021-01-20T23:52:15.513Z",
                "org.opencontainers.image.description": "Container and system event tracing using eBPF",
                "org.opencontainers.image.licenses": "Apache-2.0",
                "org.opencontainers.image.revision": "3aa104c8bca6027e3807e0ac86e1d3cdd90bddd7",
                "org.opencontainers.image.source": "https://github.com/xxx/tracee",
                "org.opencontainers.image.title": "tracee",
                "org.opencontainers.image.url": "https://github.com/xxxx/tracee",
                "org.opencontainers.image.version": "main-test"
            }
```

**Comment by itaysk** [comment]
closed by #625
